overrides/windows-override.json

```diff
@@ -1177,7 +1177,8 @@
             }
         },
         "pinnedTabs": {
-            "state": "internal",
+            "state": "preview",
```

Comment:
Are you planning an incremental rollout of pinned tabs? If so, you would need to use a sub-feature...